aiohttp/web_app.py

```diff
@@ -253,6 +253,17 @@ def make_handler(self, *,
                       access_log_class=access_log_class,
                       loop=self.loop, **kwargs)
 
+    def make_handler(self, *,
+                     loop=None,
+                     access_log_class=AccessLogger,
+                     **kwargs):
+        warnings.warn("@make_handler is deprecated",
```

Comment:
Please replace `@make_handler` with `Application.make_handler(...)` and add a test for making sure that deprecation warning is raised.